js/plot/util/scales.js

```diff
@@ -86,7 +86,10 @@ var scales = function(opts) {
       if ((!data) || (data.length === 0)) {
         return [];
       }
-      var defaultTicks = opts.bgUnits === 'mg/dL' ? [40, 80, 120, 180, 300] : [2.0, 4.5, 7.0, 10.0, 16.0];
+      var defaultTicks = _.map(_.values(_.omit(opts.bgClasses, 'very-high')), function(n) {
+        return _.get(n, 'boundary');
+      }).sort(function (a, b) { return a - b; });
```

Comment:
Yeah, after some digging it would be a prohibitively large amount of work to alter the label placement outside of `d3.svg.axis`, so we're just going to drop the `very-low` at least till we port the daily view to `viz`